### PR TITLE
duck 4.8.4.20017

### PR DESCRIPTION
**Pull request by G-Square**
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

_You can erase any parts of this template not applicable to your Pull Request._

### New Formulae Submissions:

- [ ] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [ ] Have you built your formula locally prior to submission with `brew install <formula>`?

### Changes to Homebrew's Core:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031) if you'd like one.
- [ ] Have you successfully ran `brew tests` with your changes locally?

### CODE REVIEW

**Comment by G-Square** [comment]
Not sure if it should duck-src-4.7.7.18995.tar.gz or 4.8.18316.
Version number and time-stamp do not agree.

File: Library/Formula/duck.rb [hunk 1]
```diff
@@ -2,9 +2,9 @@ class Duck < Formula
   desc "Command-line interface for Cyberduck (a multi-protocol file transfer tool)"
   homepage "https://duck.sh/"
   # check the changelog for the latest stable version: https://cyberduck.io/changelog/
```
Comment (mikemcquaid) [inline Library/Formula/duck.rb]: Did you check this? It doesn't appear to be stable yet.

Comment (G-Square) [inline Library/Formula/duck.rb]: Yes, I checked Cyberduck.io it only list Cyberduck changes not CLI. And `https://dist.duck.sh/` appears to be a dump from somewhere else. SVN & Git is up to date. 
https://g.iterate.ch/projects/ITERATE/repos/cyberduck/browse

Comment (mikemcquaid) [inline Library/Formula/duck.rb]: I'm going to defer to @dkocher here.

**Comment by UniqMartin** [comment]
@G-Square Please wait with pushing more commits to this branch; we're having some issues with our CI and doing that won't magically resolve things. Aside from that, can comment on why you are trying all kinds of different versions in your commits? What is the version you're actually trying to update this to?

**Comment by G-Square** [comment]
Ok I am halting testing. Goal was to update to `4.7.7` or `4.8` (if stable). 
I was trying lower versions to see if any would work and examining error difference. 
However after testing `origin/master` version of `4.7.3` and failing to build this confirms CI issues.

Thanks

File: Library/Formula/duck.rb [hunk 2]
```diff
@@ -14,8 +14,9 @@ class Duck < Formula
   end
 
   depends_on :java => ["1.8+", :build]
-  depends_on :xcode => :build
+  depends_on :xcode => ["7.1+", :build]
```
Comment (G-Square) [inline Library/Formula/duck.rb]: Prerequisites from [Git Cyberduck](https://g.iterate.ch/projects/ITERATE/repos/cyberduck/browse)

**Comment by apjanke** [comment]
Those CI issues may be sorted out now (see #50651).

PRs should be created from a dedicated branch, not your `master` branch. That's why a bunch of extraneous commits got pulled in to this PR. PRs from `master` are hard to work with.

Why don't you close this one out and submit a new PR from a dedicated branch, for `duck 4.8.4`. There's a good chance it'll work now.

**Comment by dkocher** [comment]
See  #49464 why the current formula is outdated.

**Pull request closed without merging**